app/(tabs)/(home)/index.tsx

```diff
@@ -74,53 +64,9 @@ export default function HomeScreen() {
       });
       setLoading(false);
     };
-    prepare();
+    void prepare();
   }, [activeTour]);
 
-  const requestPermissionsAsync = async () => {
-    return await Notifications.requestPermissionsAsync({
-      ios: {
-        allowAlert: true,
-        allowBadge: true,
-        allowSound: true,
-      },
-    });
```

Comment:
Kannst du das in den prepare reinmachen, sonst bekommt der Nutzer keine Benachrichtigungen angezeigt